Sasquatch/SasquatchObjC/Info.plist

```diff
@@ -35,7 +35,11 @@
 	<true/>
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>Select photo for attachment</string>
-	<key>UIBackgroundModes</key>
+    <key>NSLocationAlwaysUsageDescription</key>
+    <string>Will you allow this app to always know your location?</string>
```

Comment:
Maybe change "will" to "do" for consistency?

Reply:
Maybe "Do you allows this app to always know your location even when not in use?"